include/posix/time.h

```diff
@@ -69,6 +69,8 @@ struct itimerspec {
 #define TIMER_ABSTIME 4
 #endif
 
+s64_t _timespec_to_timeoutms(const struct timespec *abstime);
```

Comment:
Generic note: not ideal place for this func (this is official POSIX header!), but as _ts_to_ms() below is already there, ok.